mappings/net/minecraft/block/FlowingFluidBlock.mapping

```diff
@@ -1,36 +1,86 @@
 CLASS net/minecraft/class_200 net/minecraft/block/FlowingFluidBlock
-	METHOD method_834 (Lnet/minecraft/class_99;IIII)I
+	FIELD field_897 adjacentFluidBlocksXZ I
+		COMMENT A counter that can have a value between 0 and 4.
+		COMMENT It stores the amount of adjacent fluid blocks along the X and Z direction.
+	FIELD field_9383 distanceToSlope [I
+		COMMENT The distance in direction 0 to 4 to the nearest slope.<br>
+		COMMENT <ul>
+		COMMENT <li>Direction 0: (x-1, y, z)</li>
```

Comment:
Maybe it's just me, but I think it'd be clearer if these would just say `Direction 0: -x (west)` etc. rather than giving an entire vector.

Reply:
yes, I was gonna do that but euhm, I was too lazy to look up which direction corresponded to which vector, but I'll update the jdocs

Reply:
Changes are now implemented!